src/transformers/agents/tools.py

```diff
@@ -185,7 +185,7 @@ def save(self, output_dir):
             "tool_class": full_name,
             "description": self.description,
             "name": self.name,
-            "inputs": str(self.inputs),
```

Comment:
This fixes a bad behaviour where the "inputs" key in the config json would be changed from dict to string, which then prevented from reading them correctly when using `load_tool`.